pkgs/development/libraries/libime/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, fetchurl
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, boost
+, python3
+, fcitx5
+}:
+
+let
+  table = fetchurl {
+    url = "https://download.fcitx-im.org/data/table.tar.gz";
```

Comment:
Will this change over time and cause a hash mismtach?

Reply:
I don't think it will. It hasn't been changed for 10 years.